.env.example

```
@@ -19,3 +19,12 @@ PBW_PATH_PREFIX=""
 # Your timezone, this impacts logging, backup filenames and default timezone
 # in the web interface.
 TZ=""
+
+# Enable or disable Prometheus metrics support.
+PBW_ENABLE_METRICS="false"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)